docs/explanation/logical-axiomatization.md

```diff
@@ -0,0 +1,101 @@
+- This is also too loose as there are cells in the central nervous system that are not neurons (eg glial cells).
+
+In such cases, sometimes not having an equivalent class axioms is better (like in the case of neuron), and asserting is the best way to classify a child. 
+
+## Style guide
+
+Each ontology has certain styles and conventions in how they axiomatize. This style guide is specific to OBO ontologies. We will also give reasons as to why we choose to axiomatize in the way we do. However, be aware of your local ontology's practices. 
+
+### Respect the ontology style
+
+As a preamble (and to avoid over prescribing style), it is important for us to note that ontologies have specific styles to things like what relation to use, and should be respected. This is usually important for their use cases. For example the cell ontology has a [guide to what relations to use](https://obophenotype.github.io/cell-ontology/relations_guide/). An example of an agreement in the community is that while anatomical locations of cells are recorded using `part of`, neurons should be recorded with `has soma location`. This is to accommodate for the fact that many neurons have long reaching synapses that cover multiple anatomical location making them tricky to axiomatize using `part of`. 
+
+For example, `Betz cell`, a well known cell type which defines layer V of the primary motor cortex, synapses lower motor neurons or spinal interneuron (cell types that reside outside the brain). Having the axiom `'Betz cell' part_of 'cortical layer V'` is definitively wrong. In this case `has soma location` is used. Because of cases like these that are common in neurons, all neurons in CL should use `has soma location`.
+
+### Avoid redundant axioms 
+
+Do not add axioms that are not required. If a parent class already has the axiom, it should not be added to the child class too. 
+For example:
+- `retinal bipolar neuron` is a child of `bipolar neuron` 
+- `bipolar neuron` has the axiom `'has characteristic' some 'cortical bipolar morphology'`
+- Therefore we do not add `'has characteristic' some 'cortical bipolar morphology'` to `retinal bipolar neuron`
+
+Axioms add lines to the ontology, resulting in larger ontologies that are harder to use. 
+
+### Let the reasoner do the work
+
+Asserted is_a parents do not need to be retained as entries in the 'SubClass of' section of the Description window in Protege if the logical definition for a term results in their inference.
+
+For example, `cerebral cortex GABAergic interneuron` has the following logical axioms:
+```
+Equivalent_To
+  'GABAergic interneuron' and 
+  ('has soma location' some 'cerebral cortex')
+```
+We do not need to assert that it is a `cerebral cortex neuron`, `CNS interneuron`, or `neuron of the forebrain` as the reasoner automatically does that. 
+
+We avoid having asserted subclass axioms as these are redundant lines in the ontology which can result in a larger ontology, making them harder to use.
```

Comment:
Already mentioned in line 79?

Reply:
repetition legitimizes, repetition legitimizes, repetition legitimizes :p 
I remember this, added it here as it is a slightly different case than that of line 79, same reason to leave out the axiom, but yeah... didn't think it was any harm to repeat this here (which is ironic given what its saying hahaha, but I think it has a useful purpose here)

Reply:
("repetition legitimizes" - in documentation as in [music](https://www.youtube.com/watch?v=rm9MhGiDGP4)? but in code & ontologies it just adds redundancy that makes maintenance difficult as you need to fix in multiple places)

Reply:
was totally channeling Adam Neely there :p